src/components/LessonCard/LessonCard.tsx

```diff
@@ -108,61 +108,147 @@ const Paragraph = styled(P)`
   line-height: 19px;
 `;
 
+const Hr = styled.hr`
+  margin: var(--spacing-x-large) 0 var(--spacing-x-large) 0;
+`;
+
 const StyledButton = styled(Button)``;
 
+const selectMascot = (mascotType: MascotType, progress?: Progress) => {
+  let maskHead = false;
+  let maskBody = false;
+  let maskLegs = false;
+  if (progress) {
```

Comment:
This has been confusing to me - what is this masking logic actually for? Is this about rendering the mascot only half way according to the progress? 

If so - while the implementation is neat, but I'm not sure where that requirement comes from as I can't see that anywhere on the design guides we have been given? I don't see it in the figma nor the prototype. Have I missed it somewhere - could you point to that?

Reply:
https://lucky-smakager-43f6a1.netlify.app/#components-lesson-cards
"The Lesson Cards use an inline SVG image with names zones of Head, Body and Legs, each SVG also has a white merged version of each part named head-bg, bogy-bg and legs-bg.
We target the visibility of these parts by passing a data attribute ( data-progress="0") of 1, 2 or 3."

This requirement is why I had originally put the SVG assets in LessonCard.tsx. That's honestly the cleanest implementation, without awkward `position`ing in the css.

Reply:
Oh interesting, I had completely missed that! I am not sure why they didn't reflect that in the figma either!

Fair enough. Anyway with that in mind, I think your current implementation is cool though. I do think it's cleaner this way with the separate assets, and I can see the mask overlay logic clearly too. Thanks 👍